### PR TITLE
Improve integrator spreadsheets

### PR DESCRIPTION
**Pull request by moritzcopter**
- add support for circle integrator spreadsheets - the mappings Circle -> taggings are defined in `config.json`
- post a notification in the integrator mattermost chat when the sheets are updated
- move code to main module to make it executable
- a few smaller improvements/fixes (deduplicate members by email, add missing municipalities)

I am working with Dan so that we can deploy this version - I am currently planning to run a weekly cronjob.

### CODE REVIEW

File: rebel_management_utilities/config/config.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "local_groups": {
```
Comment (martijnmartijnmartijnmartijn) [inline rebel_management_utilities/config/config.json:2]: Last weekend I updated the names of the municipalities that we use to the official names used by the government. I also updated people municipality field accordingly, so no one now has the old names. Some of the names do occur in file still, so could you change:

"Den Haag" to "'s-Gravenhage"
"Bergen (Limburg)" to "Bergen (L.)"
"Bergen (Noord Holland)" to "Bergen (NH.)"
"Capelle" to "Capelle aan den IJssel"
"Nuenen c.a." to "Nuenen, Gerwen en Nederwetten"

And could you remove:
"Dantumadeel" (this is the old spelling, the new one is "Dantumadiel" but is already in the file)
"De Friese Meren" (same as above - the new spelling "De Fryske Marren" is already in the file)
"Peel" (this is a non-existent municipality, the actual one is "Peel en Maas")

**Comment by martijnmartijnmartijnmartijn** [comment]
Don't have any other comments on the code - looks good, and since we have already seen it in action I know it works.

File: rebel_management_utilities/config/config.json (hunk 1 above)
Comment (moritzcopter) [reply rebel_management_utilities/config/config.json:2]: Thanks, I updated those

File: rebel_management_utilities/update_integrator_spreadsheets.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    with open('resources/signups_message.md', 'r') as f:
+        message = f.read()
+
+    message = message.format(total_signups=total_signups, signup_table=df_grouped.to_markdown(index=False))
+
+    post_to_channel(LOCAL_GROUP_INTEGRATORS_CHANNEL_ID, message)
+
+
+if __name__ == "__main__":
+    start_date = datetime.date.today() - datetime.timedelta(days=7)
```
Comment (martijnmartijnmartijnmartijn) [inline rebel_management_utilities/update_integrator_spreadsheets.py:50]: The idea is to automatically run this script weekly right? I think we could run into problems when we somehow delay running this script by some number of days. Then we take only rebels from the past 7 days, so we miss rebels from in between. Maybe a fix can be to save the previous data of querying in a file, load that, and always continue from then on?

Comment (martijnmartijnmartijnmartijn) [reply rebel_management_utilities/update_integrator_spreadsheets.py:50]: Other than that no issues, so if this is fixed I'm think we're good to merge

Comment (moritzcopter) [reply rebel_management_utilities/update_integrator_spreadsheets.py]: Thats fair - I think caching locally does not always work as we might run the script from different places (e.g. scheduled from the server but sometimes locally if we want to backfill some specific group). I now made the lookback time configurable which should give us enough flexibility

**Review by martijnmartijnmartijnmartijn — APPROVED** [review]

**Pull request merged**